### DISCUSSION TITLE
Welcome to cucumber-ts-starter Discussions!

### DISCUSSION THREAD

**Discussion by hdorgeval** [hdorgeval/cucumber-ts-starter#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ddanilovic5** [comment]
Hey guys,

I'm not able to debug code...
After npm install, I added @debug tag before scenario and set break points in _simple-maths-steps.ts_ file.

![image](https://user-images.githubusercontent.com/72069842/104755174-6bab6580-575a-11eb-9970-e262a3ed3740.png)
![image](https://user-images.githubusercontent.com/72069842/104755269-841b8000-575a-11eb-9197-c1e2fb286571.png)

Tests were executed without stopping at any breakpoints.

Any ideas what can be a problem?

Thanks,

Dusan

**Reply by hdorgeval** [reply]
Hi @ddanilovic5 , you should open the `Run` panel and click on  the `Launch Features`:

![Screen Shot 2021-01-15 at 19 11 56](https://user-images.githubusercontent.com/9698924/104763575-65bb8180-5766-11eb-8390-80be92d58dc1.png)

**Reply by ddanilovic5** [reply]
Yap, that's it. Thanks!

**Comment by hdorgeval** [comment]
Hi @ddanilovic5 , you have also a keyboard shortcut to start the debug session: `F5`
